docs/rst/user_manual/commands/topic.rst

```diff
@@ -30,8 +30,7 @@ The output format is as follows: :ref:`user_manual_command_topic_output_simple`.
 Verbose
 -------
 
-This argument queries for more complete information about each of the topics in the network.
-It adds the Guid of each endpoint on the topic and the whether the type has been discovered.
```

Comment:
Why was this removed? Still holds right?

Reply:
The guid is only for the higher verbosity mode

Reply:
So we are only documenting what information is provided through examples? Maybe it should be explained what extra information is showed in verbose mode, and in super verbose mode.